### PR TITLE
feat: fix responsiveness

### PR DESCRIPTION
**Pull request by shlomomdahan**
Fixing some responsiveness issues I noticed.

### CODE REVIEW

File: src/components/StatsInDetail/Charts/OverallTrendsChart.tsx [hunk 1]
```diff
@@ -54,30 +55,34 @@ const Chart: React.FC<ChartProps> = ({ csvPath, title, width = '100%', height =
...
+                axisTick: { show: true, alignWithLabel: true },
             },
             yAxis: {
                 type: 'value',
                 splitLine: { lineStyle: { type: 'dashed' } },
             },
             series: [
                 {
-                    data: chartData.values,
+                    data: chartData.dates.map((date, index) => [date, chartData.values[index]]),
```
Comment (lemeurherve) [inline src/components/StatsInDetail/Charts/OverallTrendsChart.tsx:75]: Is this related to responsiveness? If not I'd suggest removing this change from this PR and create another one.

Same for the areaStyle below.

Comment (krisstern) [reply src/components/StatsInDetail/Charts/OverallTrendsChart.tsx:75]: Should heed this suggestion moving forwards, but I suspect the plot would not render properly without this, so will make an exception for it this time.

**Review by krisstern — APPROVED** [review]
